app/src/main/java/com/zulip/android/activities/ZulipActivity.java

```diff
@@ -259,10 +259,31 @@ public void muteTopic(Message message) {
 
     @Override
     public void recyclerViewScrolled() {
-        if (chatBox.getVisibility() == View.VISIBLE && !isTextFieldFocused) {
+
+        if(chatBox.getVisibility()== View.VISIBLE && !isCurrentModeStream()){
+            //check if messageEt is empty or not
+            if(messageEt.getText().toString().equals(""))
+            {
+                displayChatBox(false);
+                displayFAB(true);
+
+            }
+        }
+
+        else if(chatBox.getVisibility()==View.VISIBLE && isCurrentModeStream()){
+            //check if messageEt is empty or not
```

Comment:
Hi! I see that line # 264 and line # 274 are the same comment; is that intentional?